scripts/sct_get_centerline.py

```diff
@@ -44,7 +44,7 @@ def get_parser():
 
     parser.add_option(name='-centerline-algo',
                       type_value='multiple_choice',
-                      description='Algorithm for centerline fitting. Only relevant with -angle-corr 1.',
+                      description='Algorithm for centerline fitting. Only relevant with -method fitseg',
```

Comment:
Out of curiosity: Is angle-corr 1 and fitseg equivalent? Not sure to understand the change.

Reply:
`-angle-corr` flag does not exist for this function